### ISSUE TITLE
Stop command uses wrong path

### ISSUE THREAD

**Issue by svensp** [ipunkt/rancherize#47]
Rancherize stop uses a double `//` when doing the stop command.
```shell
rancherize start local
  RUN  'docker-compose' '-p' 'PROJECT' '-f' './.rancherize/docker-compose.yml' 'up' '-d'
  RES  Command ran successfully
rancherize stop local
  RUN  'docker-compose' '-p' 'PROJECT' '-f' './.rancherize//docker-compose.yml' 'stop'
  RES  Command ran successfully
```

**Comment by svensp** [comment]
Reported by rokde

**Comment by svensp** [comment]
Fixed in master and going to be in 2.18.0
